### DISCUSSION TITLE
default forder for FilePicker dialogbox

### DISCUSSION THREAD

**Discussion by TranquilleXav** [MicrosoftEdge/WebView2Feedback#2748] [Q&A]
Hello
Is there a way to set the default folder for opening the FilePicker dialog box?

when webview2 profile is new, filepicker opens in documents folder
when i browse ans select file, the last folder is memorized. 
where is it stored?
I did not find a file containing this information...

and when i force values in registry  (via LastVisitedPidlMRU and other keys, I have used this technique before for IE11 and it worked fine), settings in not working with webview2...

thank you in advance

**Comment by victorthoang** [comment]
Hello @TranquilleXav,

I will turn this into an issue so that we can have one of our devs look into your question. Generally we use the Issues page and assign it with the "Question" tag to have it looked at. The Discussions tab is more for off topic or non-issue related topics!

**Reply by victorthoang** [reply]
https://github.com/MicrosoftEdge/WebView2Feedback/issues/2797

**Reply by TranquilleXav** [reply]
Hello @victorthoang 
ok thank you for your help!
